src/commands/force/source/beta/status.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+import { FlagsConfig, flags, SfdxCommand } from '@salesforce/command';
+import { Messages } from '@salesforce/core';
+
+import {
+  SourceTracking,
+  throwIfInvalid,
+  replaceRenamedCommands,
+  getKeyFromObject,
+  getKeyFromStrings,
+  ChangeResult,
+} from '@salesforce/source-tracking';
+
+import { StatusResult, StatusFormatter } from '../../../../formatters/statusFormatter';
+
+Messages.importMessagesDirectory(__dirname);
+const messages: Messages = Messages.loadMessages('@salesforce/plugin-source', 'status');
+
+export default class SourceStatus extends SfdxCommand {
+  public static description = messages.getMessage('description');
+  public static readonly examples = replaceRenamedCommands(messages.getMessage('examples')).split(os.EOL);
+  protected static flagsConfig: FlagsConfig = {
+    all: flags.boolean({
+      char: 'a',
+      description: messages.getMessage('flags.all'),
+      longDescription: messages.getMessage('flags.allLong'),
+    }),
```

Comment:
I didn't even know this flag existed on the legacy command.  What's the point?  Don't we default to all if no flags are passed?

Reply:
Yes.  I think it's a stupid flag and I'd love to get rid of it.